lib/src/utilities.dart

```diff
@@ -363,6 +373,10 @@ class NormalisedInputType {
     if (sessionExpiredStatusCode != null)
       _sessionExpiredStatusCode = sessionExpiredStatusCode;
 
+    var _maxRetryAttemptsForSessionRefresh = 10;
+    if (maxRetryAttemptsForSessionRefresh != null)
+      _maxRetryAttemptsForSessionRefresh = maxRetryAttemptsForSessionRefresh;
```

Comment:
add brackets

Reply:
Added for all the if conditions in that file.